### ISSUE TITLE
months should be 3-letter macros

### ISSUE THREAD

**Issue by cpmpercussion** [NIME-conference/NIME-bibliography#34]
some months are literals (e.g., `{"April"}`) and some use the 3-letter macros (jan, feb, mar, apr, may, jun, jul, aug, sep, oct, nov, dec).

3-letter macros are [preferred](http://tug.ctan.org/tex-archive/biblio/bibtex/contrib/doc/btxFAQ.pdf) because whatever bibstyle will turn them into the correct month name (in the correct language) when required.

so let's chip away at this over all the proceedings.

**Comment by alexarje** [comment]
Yes, good. I am not really sure if it is really necessary to include the month? Again, this is probably more relevant for some journals than for an annual conference.

**Comment by cpmpercussion** [comment]
I don't think it's super necessary either, but I guess it's there if people want to know 😄 

actually it turns out to be a bit tricky to preserve the 3-letter macros with `bibtexparser`, so maybe better to go with the flow to use full months or just drop the month from the entries.

**Comment by alexarje** [comment]
I would skip the month. From a historical perspective, it would have been more interesting to have the conference dates in there, but I don't think that is very important either. Saving some bytes in the file is probably also nice from a parsing perspective.
